roles/taca/tasks/main.yml

```diff
@@ -26,17 +26,28 @@
     site_full: "uppsala"
     ngi_pipeline_site_path: "{{ ngi_pipeline_upps_path }}"
     ngi_pipeline_site_delivery: "{{ ngi_pipeline_upps_path }}"
+    proj_path: "/proj/"
+    extra_reports:"     -\n        - <ANALYSISPATH>/qc_sisyphus/reports\n        - <STAGINGPATH>/00-Reports/SequenceQC/Sisyphus"
   tags: taca
 
 - name: Deploy upps_taca_delivery.yml config
   template: src="site_taca_delivery.yml.j2" dest="{{ ngi_pipeline_conf }}/TACA/{{ site }}_taca_delivery.yml"
   tags: taca
 
+- set_fact:
+    proj_path: "/pica/sw/links/proj/"
```

Comment:
should there be a `tags: taca` statement here as well (and after line 62 below)?